src/javascripts/jquery.tipsy.js

```diff
@@ -118,6 +118,21 @@
                 }
             }
         },
+
+        destroy: function(){
+          $.removeData(this.$element.get(0));
```

Comment:
You're right @joscha, `this.$element.removeData()` would also work.. I was following suits with this `$.data(ele, 'tipsy', tipsy);` in `$.fn.tipsy` and didn't even think about using .removeData() on the jQ-element :)